packages/app-admin/src/plugins/init.tsx

```diff
@@ -4,6 +4,7 @@ import { registerPlugins, getPlugins } from "@webiny/plugins";
 import { ReactComponent as SettingsIcon } from "@webiny/app-admin/assets/icons/round-settings-24px.svg";
 import { WebinyInitPlugin } from "@webiny/app/types";
 import { AdminMenuSettingsPlugin } from "@webiny/app-admin/types";
+import { hasScopes } from "@webiny/app-security"; 
```

Comment:
The Admin app should not depend on the `app-security` app.

Reply:
Instead of relying on the `hasScopes` function and the `scopes` property you've added here (not a bad idea), maybe we could just see if any of the registered plugins have returned something.

In other words, can you check... is it possible to determine the `canSeeAnySettings` by examining the output of all registered plugins? 

If all are `null`, maybe we can consider that as an indicator that we don't have to render the Settings item at all.

What do you think @TommyJackson85 ?

Reply:
For sure I can look into it.
Just for further context, what would be the purpose of doing this? Is it for better performance and speed? Just asking for learning purposes.
I did use hasScopes for for all welcome widgets in ```app-plugin-admin-welcome-screen``` to see if a separte message needs to be displayed. Do you think we would do the same to the welcome widgets? Maybe create a new issue for it?

Reply:
First, I wanted to try to avoid adding additional properties to the plugin, and keep it as simple as possible. The other, more important thing, is that we don't want to have security-app functionality in the admin-app. The admin-app should be as standalone as possible, and introducing a security-app related prop here would break that. Ideally, if you wanted to just use the admin-app for creating a simple admin interface, we should be able to do it.

Can you give a reference or two regarding the `app-plugin-admin-welcome-screen` ?

Reply:
@doitadrian 
On an unauthorised account, and from running this is in the console..
```const settingsPlugins = getPlugins<AdminMenuSettingsPlugin>("admin-menu-settings");```
I get:
<img width="352" alt="Screenshot 2020-07-16 at 20 46 14" src="https://user-images.githubusercontent.com/22279028/87715587-7bdf8980-c7a5-11ea-806f-e2e7e7fe0396.png">

Doesn't seem to be anything displaying. What about doing an individual ```hasScopes``` security check for each one from each respective file there I initially passed the scope then block the plugin from being passed through?
OR maybe from the init function we could filter them out based on boolean values (i.e. a potential key passed called permitted to use.)